app/src/main/java/app/revanced/manager/ui/screen/PatchesSelectorScreen.kt

```diff
@@ -101,16 +101,17 @@ fun PatchesSelectorScreen(
             )
         },
         floatingActionButton = {
-            ExtendedFloatingActionButton(
-                text = { Text(stringResource(R.string.patch)) },
-                icon = { Icon(Icons.Default.Build, null) },
-                onClick = {
-                    composableScope.launch {
-                        // TODO: only allow this if all required options have been set.
```

Comment:
This comment refers to patch options, which hasn't changed here, so the comment should be kept.